package.json

```diff
@@ -10,7 +10,9 @@
     "undo": "npx sequelize-cli db:migrate:undo",
     "seed": "npx sequelize-cli db:seed:all",
     "undo-seed": "npx sequelize-cli db:seed:undo",
-    "test": "npm run migrations && npm run seed && nyc mocha --require @babel/register ./tests/*.js  --timeout 25000 --exit && npm run undo-seed && npm run undo",
+    "pretest": "npm run migrations && npm run seed",
+    "test": "nyc mocha --require @babel/register ./tests/**/*.js  --timeout 30000 --exit",
```

Comment:
I wonder why you changed these scripts of test, and how to run the test after you change them. You should document this change so that the team knows a new way of testing